### PR TITLE
Testnet Competition Code Changes

### PR DESCRIPTION
**Pull request by noahlitvin**
Notes to auditors :
- Changed `getCollateralPrice` to take in an amount.  This will ensure when synths need prices, the amount can be sent into the oracle node to determine price. (Changed by **sunny**)
- Removed `checkPriceDeviation` from order settlements on both spot/perps markets.
- Perps market settlement calls `getUpdateFee` first prior to calling `parseUpdatePriceFeeds`

### CODE REVIEW

**Comment by noahlitvin** [comment]
Released as `3.3.3-dev.32324850`

**Comment by noahlitvin** [comment]
Released as `3.3.3-dev.05f72e70`

**Review by dbeal-eth — APPROVED** [review]
good.

File: protocol/synthetix/contracts/storage/Pool.sol [hunk 1]
```diff
@@ -476,7 +479,7 @@ library Pool {
     ) internal view returns (uint256 collateralAmountD18, uint256 collateralValueD18) {
         uint256 collateralPriceD18 = CollateralConfiguration
             .load(collateralType)
-            .getCollateralPrice();
+            .getCollateralPrice(DecimalMath.UNIT);
```
Comment (bachstatter) [inline protocol/synthetix/contracts/storage/Pool.sol:482]: So we do not want to value the users collateral based on amounts?

**Pull request merged**